src/test/engine-integrations/cross-margin/MintTokenWithPartialMarginBeta.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// import test base and helpers.
+import {CrossMarginFixture} from "./CrossMarginFixture.t.sol";
+
+import "../../../config/enums.sol";
+import "../../../config/types.sol";
+import "../../../config/constants.sol";
+import "../../../config/errors.sol";
+
+import "../../../core/engines/cross-margin/types.sol";
+
+import "../../../test/mocks/MockERC20.sol";
+
+// solhint-disable-next-line contract-name-camelcase
+contract TestMintWithPartialMarginBeta_CM is CrossMarginFixture {
+    MockERC20 internal lsEth;
+    MockERC20 internal mmc;
```

Comment:
what does lsEth and mmc stands for?

Reply:
LsETH - liquid staked eth
mmc - money market type coin

Reply:
better add them somewhere if you haven't

Reply:
What do you mean?

Reply:
just meant comment them somewhere in the code